src/backend/utils/utils.py

```diff
@@ -0,0 +1,24 @@
+import random
+import string
+
+
+def random_label(prefix="", *extra_prefixes, length=10):
```

Comment:
Take extra_prefixes as a list. And return only one single symbol with these prefixes in order

Reply:
Done